projects/sandbox/vizapp/vizapp/pages/performance_summary/sensitive_volume.py

```diff
@@ -75,25 +84,37 @@ def initialize_sources(self):
             # (10, 10)
         ]
         self.color_map = {i: j for i, j in zip(mass_combos, palette)}
-        self.color_map["MDC"] = "#000000"
+        self.color_map["End O3"] = "#000000"
 
         foreground = self.page.app.foreground
         rejected = self.page.app.rejected_params
 
-        self.probs = {}
+        self.weights = {}
         cosmology = self.page.app.cosmology
         for combo in mass_combos:
-            prior, _ = log_normal_masses(*combo, sigma=1, cosmology=cosmology)
+            # TODO: make sigma input parameter of app
+            self.logger.info(f"Computing likelihoods under {combo} log normal")
+            prior, _ = log_normal_masses(
+                *combo, sigma=0.1, cosmology=cosmology
+            )
             prob = get_prob(prior, foreground)
             rejected_prob = get_prob(prior, rejected)
-            self.probs[combo] = (prob, rejected_prob)
+
+            weight = prob / self.source_probs
+            rejected_weights = rejected_prob / self.source_rejected_probs
+            norm = weight.sum() + rejected_weights.sum()
+            self.weights[combo] = weight / norm
 
         self.line_source = ColumnDataSource(dict(x=self.x))
 
         band_x = np.concatenate([self.x, self.x[::-1]])
         self.band_source = ColumnDataSource(dict(x=band_x))
         self.update()
 
+    def get_err(self, V, dV):
+        denom = np.sqrt(4 * np.pi) * 3 * V
```

Comment:
This is correct as-is, but I think it's clearer if it's phrased as `denom = 4 * np.pi * distance ** 2` (like, making it explicitly 1 / (dV/dr)).

Reply:
Ahhh wow that's a much cleaner reformulation, great catch. Let me know how the rewrite looks and then I'll merge